### PR TITLE
fixes https://github.com/enclave-networks/docs/issues/73

### PR DESCRIPTION
**Pull request by enclave-marc-barry**
changes to the windows installation instructions to surface 32-bit installer links and make installation instructions clearer

### CODE REVIEW

File: docs/setup/windows.md [hunk 1]
```diff
@@ -61,101 +62,148 @@ Users should remove configuration and private keys when uninstalling if they are
...
+## Unattended installation
 
-If an install goes horribly wrong, remove Enclave using the uninstaller and try again. If you're still encountering problems, examine the log files in `C:\Program Files\Enclave Networks\Enclave\Agent\logs`. If an install is interrupted use the Enclave setup file to restart the process.
+Perform a silent installation of Enclave without a UI by using the unattended installer. The unattended installer is optimised for deployment via Group Policy, Configuration Manager or with other automated deployment tooling. We recommend using unattended installers for bulk distribution and silent installation.
 
-For troubleshooting and errors, use the site search or visit our [troubleshooting](/kb/#troubleshooting) section to look for information about common error messages.
+For most unattended installation scenarios, you should use the 64-bit executable (.exe) installer.
 
-If your installation fails and you are unable to resolve the problem, please contact <a href="mailto:support@enclave.io">support@enclave.io</a>.
+=== "64-bit executable"
```
Comment (enclave-marc-barry) [inline docs/setup/windows.md:71]: Added tabs

File: docs/setup/windows.md [hunk 2]
```diff
@@ -61,101 +62,148 @@ Users should remove configuration and private keys when uninstalling if they are
...
+        ```
+
+        Providing an invalid enrolment key will cause the installation to fail (and roll back).
+
+        !!! note
+              If your tooling provides an enrolment key to the installer, at the time of installation the target system will require an Internet connection to perform the enrolment.
+              
+              If you need to deploy Enclave to a system without an Internet connection, use the unattended installer without the `ENROLMENT_KEY` property, then use the `enclave enrol` CLI command to enrol the system when an Internet connection becomes available.
+
+=== "64-bit MSI package"
```
Comment (enclave-marc-barry) [inline docs/setup/windows.md:147]: MSI used to be a separate section, but that doesn't make a lot of sense so I've rolled it up into the EXE tabs

File: docs/setup/windows.md [hunk 3]
```diff
@@ -1,53 +1,54 @@
 # Windows
 
-<small>This guide applies to Windows 7, 8, 10, 11 and Windows Server 2012, 2016, 2019 and 2022 (all editions). Packages are available for the x64 architecture.</small>
+## Installation
 
-!!! note
-      On Windows 7 please ensure that [Windows Management Framework 5.1](https://www.microsoft.com/en-us/download/details.aspx?id=54616) is installed before installing Enclave.
```
[the remark this answers is not in the document]
Comment (enclave-tom) [reply docs/setup/windows.md:6]: This section is still relevant for 64-bit windows 7 can we ensure that it's visible for both?

**Comment by enclave-tom** [comment]
We should probably rethink how the installation section works, as right now we show 32-bit installer then just say we don't support it then we go straight to starting and stopping? I also don't think that should be a numbered point instead a warning or note box

image

**Comment by enclave-tom** [comment]
image
This warning should be present on all 3 tabs as it's relevant to any win 7 version not just 32 bit

File: docs/setup/windows.md [hunk 4]
```diff
@@ -1,53 +1,57 @@
...
+    4. Once installed, Enclave will start and add a new tray icon to your system tray. If the Enclave tray icon is not visible, click the up arrow to find it in the hidden system tray overflow area.
 
-4. Once installed, Enclave will start and add a new tray icon to your system tray. If the Enclave tray icon is not visible, click the up arrow to find it in the hidden system tray overflow area.
+        ![/images/setup/enclave-tray-icon.png](/images/setup/enclave-tray-icon.png)
 
-    ![/images/setup/enclave-tray-icon.png](/images/setup/enclave-tray-icon.png)
+    5. Right click on the tray icon when you need to open Enclave.
 
-5. Right click on the tray icon when you need to open Enclave.
+    6. Depending on the type of enrolment key you used to enrol your new system, it might be held waiting for an Administrator to provide enrolment approval in your account [Portal](https://portal.enclave.io/). Log in to authorise the enrolment of your new system if you need to and configure additional options like DNS.
```
Comment (enclave-charlie) [inline docs/setup/windows.md:29]: "held waiting" might be a bit ambiguous for someone not familiar with our system.

"Depending on the type of enrolment key you used to enrol your new system, you might not be able to connect to other systems until an Administrator has approved your enrolment."

The last sentence feels out of place. "Log in to authorise the enrolment of your new system if you need to and configure additional options like DNS" only applies to administrators, but reads as if anyone can provide the authorisation.

File: docs/setup/windows.md [hunk 5]
```diff
@@ -1,53 +1,57 @@
...
-**You're all set!** You've successfully enrolled a new system to your Enclave account.
+    !!! warning
+        Enclave doesn't ship an **interactive installer** for 32-bit systems for compatibility reasons.
+        
+        Please use one of the 32-bit **[unattended installers](#unattended-installation)** instead.
 
-### Starting and stopping Enclave
+### Starting and stopping
 
 The Windows installer creates a lightweight supervisor service (named **Enclave** in the Services MMC snap-in) which automatically runs at system start and is responsible for starting the Enclave fabric. The supervisor service exists to start, stop and restart Enclave fabric in the background as child processes.
```
Comment (enclave-charlie) [inline docs/setup/windows.md:40]: missing a "the"?

"The supervisor service exists to start, stop and restart the Enclave fabric in the background as child processes."

File: docs/setup/windows.md [hunk 6]
```diff
@@ -1,53 +1,57 @@
...
 
 The supervisor service responds to the Enclave CLI verbs `start` and `stop` to control the Fabric.
 
 <!--- TODO The supervisor service responds to the Enclave CLI verbs [start](/handbook/fabric/cli/start) and [stop](/handbook/fabric/cli/stop) to control the Fabric. --->
 
-### Updating Enclave
+### Updating
 
 The Windows Tray application will prompt users when an updated version of Enclave is available. Following the `click here` to upgrade link in the yellow bar will terminate any Enclave connections temporarily while the setup installs the latest version.
```
Comment (enclave-charlie) [inline docs/setup/windows.md:48]: "Pressing `click here" in the yellow banner will terminate any Enclave connections temporarily while the setup installs the latest version." is a bit clearer

File: docs/setup/windows.md [hunk 7]
```diff
@@ -1,53 +1,57 @@
...
 
 Enclave should be upgraded in-place and will automatically and re-establish connectivity to peers following an update. Updating Enclave to a new version should not interrupt connectivity for more than 1 minute in total, in most cases an upgrade takes less than 30 seconds before connectivity is restored.
```
Comment (enclave-charlie) [inline docs/setup/windows.md:50]: "Updating Enclave to a new version should not interrupt connectivity for more than 1 minute in total, in most cases an upgrade takes less than 30 seconds before connectivity is restored." should be two-sentences:

Updating Enclave to a new version should not interrupt connectivity for more than 1 minute in total. In most cases, an upgrade takes less than 30 seconds before connectivity is restored.

File: docs/setup/windows.md [hunk 8]
```diff
@@ -57,105 +61,151 @@ C:\Program Files\Enclave Networks\Enclave\Agent\profiles\
...
-!!! warning
+!!! note
 
     Enclave does not backup a system's private keys. Lost or deleted private keys are not recoverable. If a system's configuration and private keys are lost, to use that system with Enclave again it must be re-enrolled.
 
-### What to do if the install fails
+## Unattended installation
 
-If an install goes horribly wrong, remove Enclave using the uninstaller and try again. If you're still encountering problems, examine the log files in `C:\Program Files\Enclave Networks\Enclave\Agent\logs`. If an install is interrupted use the Enclave setup file to restart the process.
+Perform a silent installation of Enclave without a UI by using the unattended installer. The unattended installer is optimised for deployment via Group Policy, Configuration Manager or with other automated deployment tooling. We recommend using unattended installers for bulk distribution and silent installation.
```
Comment (enclave-charlie) [inline docs/setup/windows.md:70]: I'd get rid of the "with", as it serves the same purpose as "via", and change "deployment tooling" to "tooling" to reduce repetition:

"The unattended installer is optimised for deployment via Group Policy, Configuration Manager and other automated tooling".

**Comment by enclave-alistair** [comment]
Unattended installer fixes were applied in #75, we have no evidence that the current verison of the attended installer doesn't work on x86, so closing this.

**Pull request closed without merging**